pkg/cloudprovider/providers/azure/azure_zones.go

```diff
@@ -49,6 +50,7 @@ func (az *Cloud) GetZoneID(zoneLabel string) string {
 // GetZone returns the Zone containing the current availability zone and locality region that the program is running in.
 // If the node is not running with availability zones, then it will fall back to fault domain.
 func (az *Cloud) GetZone(ctx context.Context) (cloudprovider.Zone, error) {
+	if az.UseInstanceMetadata {
```

Comment:
No standard go formatting, golint should fix it

Reply:
golint was not installed caused above issue. Now fixed.